lib/plugins/package/lib/zipService.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
       this.serverless.cli.log('Excluding development dependencies...');
 
       return BbPromise.bind(this)
-        .then(() => excludeNodeDevDependencies(servicePath))
+        .then(() => this.excludeNodeDevDependencies(servicePath))
```

Comment:
I think it'll be good enough to decorate `excludeNodeDevDependencies` with `_.memoize`

Reply:
Good point ;)